extensions/wikia/JWPlayerTag/scripts/jwplayertag.ads.js

```diff
@@ -0,0 +1,86 @@
+define('wikia.articleVideo.jwplayertag.ads', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.video.vastUrlBuilder',
+	'ext.wikia.adEngine.slot.service.megaAdUnitBuilder',
+	'ext.wikia.adEngine.slot.service.srcProvider',
+	'ext.wikia.adEngine.video.vastDebugger',
+	'ext.wikia.adEngine.video.player.jwplayer.adsTracking',
+	'wikia.log'
+], function (adContext, vastUrlBuilder, megaAdUnitBuilder, srcProvider, vastDebugger, adsTracking, log) {
+	'use strict';
+
+	var aspectRatio = 640 / 480,
+		baseSrc = adContext.get('targeting.skin') === 'oasis' ? 'gpt' : 'mobile',
+		videoPassback = 'jwplayer',
+		videoSlotName = 'VIDEO',
+		videoSource,
+		logGroup = 'wikia.articleVideo.jwplayertag.ads';
+
+	function buildVastUrl(position, videoDepth, correlator) {
+		var options = {
+				correlator: correlator,
+				vpos: position
+			},
+			slotParams = {
+				passback: videoPassback,
+				pos: videoSlotName,
+				rv: videoDepth,
+				src: videoSource
+			};
+
+		options.adUnit = megaAdUnitBuilder.build(slotParams.pos, slotParams.src);
+
+		log(['buildVastUrl', position, videoDepth, slotParams, options], log.levels.debug, logGroup);
+
+		return vastUrlBuilder.build(aspectRatio, slotParams, options);
+	}
+
+	return function(player) {
+		var correlator,
+			videoElement = player && player.getContainer && player.getContainer(),
+			videoContainer = videoElement && videoElement.parentNode,
+			prerollPositionReached = false,
+			trackingParams = {
+				adProduct: 'in-article-video',
```

Comment:
Can we set just `video` here? So it'll be consistent with N&S: https://github.com/Wikia/f2/blob/master/frontend/components/ads/slots.js#L147

Reply:
Sure :)